### PR TITLE
change to workload-insights only

### PR DESCRIPTION
**Pull request by yves-chan**
- change all workloads-shared and workload-onboarding to workload-insights

## PR Checklist

* [ ] Explain your changes, so people looking at the PR know *what* and *why*, the code changes are the *how*.

### If adding or updating templates:
* [ ] post a screenshot of templates and/or gallery changes
* [ ] ensure all steps have meaningful names
* [ ] ensure all parameters and grid columns have display names set so they can be localized
* [ ] ensure that parameters id values are unique __or they will fail PR validation__ (parameter ids are used for localization)
* [ ] ensure that steps names are unique __or they will fail PR validation__ (step names are used for localization)
* [ ] grep `/subscription/` and ensure that your parameters don't have any hardcoded resourceIds __or they will fail PR validation__
* [ ] remove `fallbackResourceIds` and `fromTemplateId` fields from your template workbook __or they will fail PR validation__

### CODE REVIEW

File: Workbooks/Workloads/Custom Telegraf/Create new profile/settings.json [hunk 1]
```diff
@@ -3,6 +3,6 @@
     "name":"Create new profile",
     "author": "Microsoft",
     "galleries": [
-        { "type": "workload-onboarding", "resourceType": "Azure Monitor", "order": 400 }
```
Comment (vutran01) [inline Workbooks/Workloads/Custom Telegraf/Create new profile/settings.json:6]: should also verify if these all shows up in the gallery the way you expect

**Review by vutran01 — APPROVED** [review]

File: Workbooks/Workloads/Collection logs/settings.json [hunk 2]
```diff
@@ -2,7 +2,6 @@
     "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/settings.json",
     "name":"View collection troubleshooting logs",
     "author": "Microsoft",
-    "galleries": [
-        { "type": "workloads-shared", "resourceType": "Azure Monitor", "order": 500 }
-    ]
+    "isPreview": true,
```
Comment (vutran01) [inline Workbooks/Workloads/Collection logs/settings.json:5]: should isPreview be false now?

**Review by vutran01 — APPROVED** [review]

File: Workbooks/Workloads/Custom Telegraf/Create new profile/settings.json (hunk 1 above)
Comment (yves-chan) [reply Workbooks/Workloads/Custom Telegraf/Create new profile/settings.json:6]: changed to just an empty gallery array, so it won't show

File: Workbooks/Workloads/Collection logs/settings.json (hunk 2 above)
Comment (yves-chan) [reply Workbooks/Workloads/Collection logs/settings.json:5]: doens't need to be since it won't show, I'll leave as is for now

**Pull request merged**